### ISSUE TITLE
Could not find a declaration file for module '@reactivex/rxjs' (5.5.x)

### ISSUE THREAD

**Issue by mdekrey** [ReactiveX/rxjs#3005]
**RxJS version:**
5.5.0, 5.5.1, 5.5.2

**Code to reproduce:**
*index.ts*
```typescript
import { Subject } from "@reactivex/rxjs";

const test = new Subject<boolean>();

test.next(true);
```

*package.json*
```json
{
  "dependencies": {
    "@reactivex/rxjs": "5.5.0",
    "typescript": "^2.5.3"
  }
}

*tsconfig.json*
```json
{
    "compilerOptions": {
        "noImplicitAny": true
    }
}
```

**Expected behavior:**
No errors.

**Actual behavior:**
> index.ts(1,25): error TS7016: Could not find a declaration file for module '@reactivex/rxjs'. 'C:/Users/mattd/Source/Examples/rxjs-bug/node_modules/@reactivex/rxjs/index.js' implicitly has an 'any' type.
>  Try `npm install @types/@reactivex/rxjs` if it exists or add a new declaration (.d.ts) file containing `declare module '@reactivex/rxjs';`

**Additional information:**
The `package.json` of rxjs contains `"typings": "./dist/package/typings/Rx.d.ts"`, but there is no `typings` folder in the `package` directory. I expect that it should be changed to `"typings": "./dist/typings/Rx.d.ts"`, because that appears to work.

**Comment by davideast** [comment]
Thanks for taking the time to file the issue @mdekrey! I confirmed this myself. The `package.json` needs to point to `"typings": "./dist/typings/Rx.d.ts",` or `typings` needs to be in `./dist/package`.

Thoughts @benlesh?

**Comment by jasonaden** [comment]
@kwonoj I don't know if we need two separate PRs for this as they are the same fix. But I sent both just to make sure we don't miss the merge on either `stable` or `master` branch.

**Comment by kwonoj** [comment]
@jasonaden I think for these kind PR we may need who merges pr manually cherrypick into master to avoid ppl have to create same pr twice, but for now for opened PR I also think it should be fine.

**Comment by jasonaden** [comment]
@kwonoj Agreed. On the Angular project we adopted certain labels so the person merging knows what to do. And we recently wrote scripts that will do rebasing based on the labels. Might want to adopt something similar. It's basically covered [in this doc](https://github.com/angular/angular/blob/master/docs/CARETAKER.md#setup) and has drastically simplified the process of merging PRs to the correct branches.
